packages/plugin-commands-import/src/import.ts

```diff
@@ -11,33 +12,56 @@ import { install, InstallOptions } from 'supi'
 import rimraf from '@zkochan/rimraf'
 import loadJsonFile from 'load-json-file'
 import renderHelp from 'render-help'
+import { parse as parseYarnLock } from '@yarnpkg/lockfile'
 
 export const rcOptionsTypes = cliOptionsTypes
 
 export function cliOptionsTypes () {
-  return {}
+  return {
+    yarn: Boolean,
+  }
 }
 
 export function help () {
   return renderHelp({
-    description: `Generates ${WANTED_LOCKFILE} from an npm package-lock.json (or npm-shrinkwrap.json) file.`,
+    description: `Generates ${WANTED_LOCKFILE} from an npm package-lock.json (or npm-shrinkwrap.json, yarn.lock) file.`,
     url: docsUrl('import'),
-    usages: ['pnpm import'],
+    descriptionLists: [
+      {
+        title: 'Options',
+        list: [
+          {
+            description: `Generate ${WANTED_LOCKFILE} from yarn.lock file`,
+            name: '--yarn',
+          },
+        ],
+      },
+    ],
+    usages: [
+      'pnpm import',
+      'pnpm import yarn',
```

Comment:
Is the option needed at all? pnpm can automatically detect whether there is a Yarn or a npm lockfile.

```suggestion
      'pnpm import --yarn',
```

Reply:
ok